test/TestUtilities/GraphicsUtil.hpp

```diff
@@ -11,6 +11,12 @@
 #include <iomanip>
 #include <limits>
 
+#ifndef SFML_OPENGL_ES
+static constexpr bool skipOpenGLESTests = false;
+#else
+static constexpr bool skipOpenGLESTests = true;
+#endif
```

Comment:
Might be clearer to instead write this like "if macro is defined, set true, else set false" rather than "if macro is not defined set false, otherwise set true". It removes an extra negative from the expression.